src/commander.ts

```diff
@@ -14,9 +14,9 @@ export class Commander {
 
     build() {
         this.extension.logger.addLogMessage(`BUILD command invoked.`)
-        if (!vscode.window.activeTextEditor || !this.extension.manager.isTex(vscode.window.activeTextEditor.document.fileName)) {
-            return
-        }
+        // if (!vscode.window.activeTextEditor || !this.extension.manager.isTex(vscode.window.activeTextEditor.document.fileName)) {
```

Comment:
This file extension check should be maintained. In the future I will examine its necessity. However it should not be in this PR.